packages/plugin-ext/src/plugin/type-converters.ts

```diff
@@ -783,6 +783,8 @@ export function fromTask(task: theia.Task): TaskDto | undefined {
     taskDto.label = task.name;
     taskDto.source = task.source;
 
+    taskDto.runOptions = task.runOptions;
```

Comment:
I am aware that this pattern is used elsewhere, but I'm not sure this is a safe assignment: this interface can be implemented by the user. Imagine the user did it like this:

``` 
export class RunOptionsImpl implements theia.RunOptions {
   private _reevaluateOnRerun;

   get reevaluateOnRerun(): boolean {
      return this._reevaluateOnRerun;
   }

   set reevaluateOnRerun(boolean doIt) {
      this._reevaluateOnRerun= doit;
   }
}
```

if we assign the object to a field in a TaskDTO and then proceed to `JSON.stringify()` the task, we'll end up with an object on the other side of the remote service which does _not_ implement `theia.RunOptions`. As far as I remember, `stringify()` does not respect property accessors.

Reply:
Good point! What should we do to mitigate that? Assign an object in this line? Because
```
    taskDto.runOptions = { reevaluateOnRerun: task.runOptions.reevaluateOnRerun };
```
should mitigate this problem, right?

Reply:
I think that should do the trick.